app/queries/servers/thread.ts

```diff
@@ -127,3 +128,9 @@ export const queryThreadsInTeam = (database: Database, teamId: string, onlyUnrea
 
     return database.get<ThreadModel>(THREAD).query(...query);
 };
+
+export const queryThreadParticipants = (database: Database, threadId: string) => {
```

Comment:
This is not use on this PR. I guess it is used on some of the other PRs and ended up here by mistake.

Fine with me if it stays, but just saying.

Reply:
This is used in both "Global threads" & "In channel" PR ... I opened this PR to have the common parts from those PRs. If it isn't clean, should I add this function in both the PRs instead?